app/views/teams/show.html.erb

```diff
@@ -18,6 +18,31 @@
   <dd><%= @team.description %></dd>
 </dl>
 
+<table class="table table-striped">
+  <tbody>
+    <% @team.members.each do |member| %>
+      <tr>
+        <td>
+          <%= member.id%> <%= member.first_name%> <%= member.last_name%>
+            
+          <% if can? :delete_ownership, @team %>
```

Comment:
- reduce "cleverness" of view by
  - do the "can user delete ownership?" check in controller
  - let controller prepare `@team.members` list to either contain an attribute (e.g. `deletable_ownership`) or have two lists: members and deletable_ownership_members (first option prefered)